doc/authoring_command_modules/authoring_commands.md

```diff
@@ -231,6 +242,31 @@ with self.command_group('mymod', mymod_sdk) as g:
     g.generic_update_command('update', supports_no_wait=True)
 ```
 
+**(5) Supporting --defer**
+
+When registering a command, the boolean `supports_local_cache` property can be used to specify that the command supports `--defer`. This will allow traditional GET and PUT requests to interact with the CLI's local object cache instead of making
+calls on the wire either for performance reasons (to avoid network latency) or because the service will only accept a payload constructed from many calls.
+
+See [Commands With Complex Types](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md#commands-with-complex-types)
+
+Here are examples:
+
+***custom_command()***
+
+```Python
+# inside load_command_table(...)
+with self.command_group('mymod', mymod_sdk) as g:
+    g.custom_command('command2', 'do_something_2', supports_local_cache=True)
+
+# inside custom.py
+def do_something_2(cmd, client, arg1, arg2, no_wait=False):
+    from azure.cli.core.commands import cached_get, cached_put
+    item = cached_get(cmd, client.get, arg1, arg2)
+    # TODO: perform some mutation of item
+    return cached_put(cmd, client.create_or_update, arg1, arg2, item)
+```
+
+Cached objects are deleted upon a successful PUT and can be view and managed using the `az cache` commands.
```

Comment:
Added a brief summary of `supports_local_cache` and a link to more detailed info in command guidance. Mentions the `az cache` group.